src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,324 @@
+date: 2020-06-03
+updated: 2020-06-03
+hero: hero.jpg
+hero_position: center
+alt: |
+  Blurry picture of traffic. Photo by Robin Pierre on Unsplash
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-6744140441987317759
+tags:
+  - blog
+  - capabilities
+  - quic
+  - networking
+  - udp
+  - webtransport
+  - behind-a-flag
+---
+
+## Background
+
+### What's QuicTransport?
+
+[QuicTransport](https://wicg.github.io/web-transport/#quic-transport) is a web API that uses the [QUIC](https://www.chromium.org/quic) protocol in a bidirectional, non-HTTP transport. It's intended for two-way communications between a web client and a QUIC server. It supports sending data both unreliably via its [datagram APIs](#datagram), and reliably via its [streams APIs](#stream).
+
+[Datagrams](https://tools.ietf.org/html/draft-ietf-quic-datagram-00) are ideal for sending and receiving data that do not need strong delivery guarantees. Individual packets of data are limited in size by the [maximum transmission unit (MTU)](https://en.wikipedia.org/wiki/Maximum_transmission_unit) of the underlying connection, and may or may not be transmitted successfully, and if they are transferred, they may arrive in an arbitrary order. These characteristics make the datagram APIs ideal for low-latency, best-effort data transmission. You can think of datagrams as [user datagram protocol (UDP)](https://en.wikipedia.org/wiki/User_Datagram_Protocol), but encrypted and congestion-controlled.
+
+The streams APIs, in contrast, provide reliable, ordered data transfer. They're [well-suited](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-streams) to scenarios where you need to send or receive one or more streams of ordered data. Using multiple QUIC streams is analogous to establishing multiple [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) connections, but QUIC streams are lightweight, and can be opened and closed without much overhead.
+
+### What's WebTransport?
+
+QuicTransport is one part of the larger [WebTransport proposal](https://wicg.github.io/web-transport/). WebTransport is a collection of APIs for sending and receiving data between a web client and a server. QuicTransport is the interface for using the QUIC protocol in the context of bi-directional WebTransport communications.
+
+Chrome is implementing the QuicTransport portion of WebTransport first, before any of the other proposed interfaces. The Chrome team made the decision to start with a QuicTransport after speaking to web developers about their use cases, and we hope to solicit early feedback on the overall WebTransport effort based on developers' experiences with QuicTransport.
+
+### Use cases
+
+This a small list of possible ways developers might use QuicTransport.
+
+- Sending game state at a regular interval with minimal latency to a server via small, unreliable, out-of-order messages.
```

Comment:
It occurs to me that you might be using 'reliable' and 'unreliable' in a specific, technical way. Do you need to define these terms some where?

Reply:
I can link to https://en.wikipedia.org/wiki/Reliability_(computer_networking)

Reply:
(I added that link to an earlier reference.)